build/installers/liferay-developer-studio/liferay-developer-studio.xml

```diff
@@ -518,7 +506,7 @@
         <parameterGroup>
             <name>dxpinfo</name>
             <title>Download Liferay DXP Bundle</title>
-            <explanation></explanation>
+            <explanation>We would not store your username and password, they are only for generating the liferay token.</explanation>
```

Comment:
Email address and password are used for generating a download token.  Your password will not be saved locally.